src/components/report/report.tsx

```diff
@@ -1,154 +1,114 @@
 import React from 'react'
 import { Box, Typography, Button } from '@mui/material'
 import { makeStyles } from 'tss-react/mui'
-import ClearIcon from '@mui/icons-material/Clear'
-import IconButton from '@mui/material/IconButton'
-import { Link } from 'react-router-dom'
 import { useNavigate } from 'react-router'
+import { CommonModal } from '../commonModal/CommonModal'
 
 const Report = () => {
   const { classes } = useStyles()
   const navigate = useNavigate()
   const handleGoBack = () => {
     navigate(-1)
   }
+  const reportReasons = [
+    { text: 'Spam', href: 'report/reportComment/Spam' },
```

Comment:
Link doesn't work. I think it should be `href: 'reportComment/Spam' `
The same for the next links.

Reply:
The links have been removed (because the design has been changed).
All the pages changes are listed in the comment: https://github.com/WeFriiends/WeFriiendsWeb/pull/58#issue-2320613876